src/data/faq_de.json

```
@@ -46,6 +46,16 @@
                             "<b>National:</b> 0800 7540002",
                             "<b>International:</b> +49 30 498 75402"
                         ]
+                    },
+                    {
+                        "title": "Wie erstelle ich ein Fehler-Protokoll?",
+                        "anchor": "error_log",
+                        "active": true,
+                        "textblock": [
+                            "Mit Version 2.2 bietet die Corona-Warn-App eine Funktion zur technischen Problem-Analyse. Das Fehler-Protokoll soll den Entwicklern helfen, schneller eine Ursachen-Analyse zu ermöglichen und Fehler effizienter zu beseitigen.<br><b>Wichtig:</b> Melden Sie sich mit einem Problem bitte <b>zuerst</b> bei der Hotline (national: 0800 7540001 bzw. international: +49 30 498 75401), um zu prüfen, ob es ggf. schon eine Lösung dafür gibt. Ein Fehler-Protokoll wird nur benötigt, wenn das CWA-Team danach fragt.",

```

Same here, maybe link to https://www.coronawarn.app/de/faq/#international_phone_numbers instead of writing the phone numbers there to be consistent with https://www.coronawarn.app/de/faq/#who_can_help?

Also, use in-app wording ("Fehlerbericht")

```suggestion
                            "Mit Version 2.2 bietet die Corona-Warn-App eine Funktion zur technischen Problem-Analyse. Der Fehlerbericht soll den Entwicklern helfen, schneller eine Ursachen-Analyse zu ermöglichen und Fehler effizienter zu beseitigen.<br><b>Wichtig:</b> Melden Sie sich mit einem Problem bitte <b>zuerst</b> bei der Technischen Hotline (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>click here to see the phone numbers</a>), um zu prüfen, ob es ggf. schon eine Lösung dafür gibt. Ein Fehlerbericht wird nur benötigt, wenn das CWA-Team danach fragt.",
```